packages/sanity/src/core/studio/components/navbar/resources/ResourcesMenuItems.tsx

```diff
@@ -0,0 +1,123 @@
+import {Box, Card, Flex, Label, MenuDivider, MenuItem, Spinner, Text} from '@sanity/ui'
+import React from 'react'
+import {SANITY_VERSION} from '../../../../version'
+import {ResourcesResponse, Section} from './helper-functions/types'
+
+interface ResourcesMenuItemProps {
+  error: Error | null
+  isLoading: boolean
+  value: ResourcesResponse | undefined
```

Comment:
small nit pick, can be also written as 
```suggestion
  value?: ResourcesResponse
```